tests/test-cases/Group6-VIC-Machine/6-3-Delete.robot

```diff
@@ -12,19 +12,20 @@ Initial load
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${container-id}=  Run And Return Rc And Output  docker ${params} create busybox
+    ${name}=  Generate Random String  15
+    ${rc}  ${container-id}=  Run And Return Rc And Output  docker ${params} create --name ${name} busybox
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${container-id}  Error
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${container-id}
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error:
-    Set Suite Variable  ${containerID}  ${container-id}
+    Set Suite Variable  ${containerName}  ${name}
     
 Delete VCH and verify
     # Get VCH uuid and container VM uuid, to check if resources are removed correctly
     Run Keyword And Ignore Error  Gather Logs From Test Server
     ${uuid}=  Run  govc vm.info -json\=true ${vch-name} | jq -r '.VirtualMachines[0].Config.Uuid'
-    Run  govc vm.power -on=true ${containerID}
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.power -on=true ${containerName}-*
     ${ret}=  Run  bin/vic-machine-linux delete --target %{TEST_URL} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource=%{TEST_RESOURCE} --name ${vch-name}
     Should Contain  ${ret}  is powered on
```

Comment:
I don't think this is reliable - given we're creating busybox without arguments it'll be running /bin/ash. Without specifying -i stdin should be closed and the container should have exited.
If that's not what's happening then it's a different behavioural problem and this test is still failure prone